python/py-build/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-build
+
+version             0.5.1
+revision            0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+python.versions     36 37 38 39
+
+maintainers         {danchr @danchr} openmaintainer
+
+description         A simple, correct PEP517 package builder
+long_description    ${description}.
+
+checksums           rmd160  3d76f057048b866a4e162792528c665e98e2be61 \
+                    sha256  16897cac845b50cca04f3c92cf8d3e9e0868b21b29b96b577333c14473baa916 \
+                    size    15243
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                        port:py${python.version}-setuptools \
+                        port:py${python.version}-wheel
+    depends_lib-append  port:py${python.version}-packaging \
+                        port:py${python.version}-pep517 \
+                        port:py${python.version}-toml
+
+    if {${python.branch} < 3.8} {
```

Comment:
the typical way here would be `if {${python.version} < 38} {`

Reply:
After a bit of digging, I think it was actually `py-python-install` that was missing a dependency. I've fixed it to use the usual style, thanks.